tpl-implementations/include/experimental/__p1673_bits/kokkos-kernels/blas2_matrix_rank_2_update.hpp

```diff
@@ -210,7 +106,7 @@ void symmetric_matrix_rank_2_update(kokkos_exec<ExecSpace> &&exec,
   const auto x_view = Impl::mdspan_to_view(x);
   const auto y_view = Impl::mdspan_to_view(y);
   auto A_view = Impl::mdspan_to_view(A);
-  mtxr2update_impl::ParallelMatrixVisitor v(ExecSpace(), A_view);
+  Impl::ParallelMatrixVisitor v(ExecSpace(), A_view);
```

Comment:
It will be troublesome in the future to have to replace all these `ExecSpace()` with the actual execution policy.  Would you consider changing `ParallelMatrixVisitor` so that it takes the execution policy directly, rather than `ExecSpace`?  That would reduce the number of places where you would have to change `ExecSpace()` to something that gets the wrapped CUDA stream out.

Reply:
This can be saved for a future PR, btw.  Thanks!